.storybook/preview.tsx

```diff
@@ -18,6 +18,12 @@ const preview: Preview = {
         date: /Date$/i,
       },
     },
+    options: {
+      storySort: (a, b) =>
+        a.id === b.id
+          ? 0
+          : a.id.localeCompare(b.id, undefined, { numeric: true }),
```

Comment:
This is dope

Reply:
This is slightly annoying bc within a story, `Docs` don't come first..

Reply:
Ehh I'll take it, I think we could also define the name of the `docs` so we could name it something like `_Docs` haha